show-land/js/script.js

```diff
@@ -14,6 +14,17 @@ const demos = [
     mainImage: "event-land-main.png", // Name of your main page image in shared/images/
     entirePageImage: "event-land-entire.png", // Name of your entire page image in shared/images/
   },
+  {
+    id: 1, // an id that is unique in the array
+    name: "SolarLand", // Name of your page
+    createdBy: {
+      name: "AbdolMajed ShahBakhsh", // Your name
+      github: "https://github.com/abdolmajed3", // link of your github profile
+    },
+    url: "solar-land/index.html", // URL of your page based on root dir
+    mainImage: "solar-land-main.png", // Name of your main page image in shared/images/
+    entirePageImage: "solar-land-entire.png", // Name of your entire page image in shared/images/
```

Comment:
comments are nice just for the first item to make it clear, you can delete the second item's comments

Reply:
ok